tests/suites/test_suite_psa_crypto.function

```diff
@@ -9783,6 +9847,59 @@ exit:
 }
 /* END_CASE */
 
+#if defined MBEDTLS_THREADING_PTHREAD
```

Comment:
Err, I guess this works by accident because `generate_test_code.py` isn't very picky, but it's neither officially supported nor necessary. `depends_on:MBEDTLS_THREADING_PTHREAD` does what was intended here and is the official way.